tt_eager/tt_dnn/op_library/composite/composite_ops.cpp

```diff
@@ -878,6 +878,23 @@ Tensor div_no_nan(
     return operation::decorate_as_composite(__func__, _div_no_nan)(input_a, input_b, output_mem_config);
 }
 
+Tensor _unary_div_no_nan(
+    const Tensor& input_a,
+    float value,
+    const MemoryConfig& output_mem_config) {
+    Tensor zero = full_like(input_a, 0.0f, output_mem_config);
+    if(value == 0)
+        return zero;
```

Comment:
Move the tensor creation into the if block.

Reply:
Done